axon-framework/monitoring-and-metrics.md

```diff
@@ -160,6 +162,8 @@ public class MetricsConfiguration {
 ```text
 # The default value is `true`. Thus you will have Metrics configured if `axon-micrometer` and appropriate metric implementation (for example: `micrometer-registry-prometheus`) are on your classpath.
 axon.metrics.auto-configuration.enabled=true
+# The default value is `false`. By enabling this property you will have message (event, command, query) payload type set as a micrometer tag/dimension by default. Additionally, the processor name is a tag/dimension now (not the part of the metrics name).
```

Comment:
```suggestion
# The default value is `false`. By enabling this property you will have message (event, command, query) payload type set as a micrometer tag/dimension by default. Additionally, the processor name will be a tag/dimension instead of it being part of the metric name.
```